src/Components/Tasks/Tasks.tsx

```diff
@@ -114,9 +116,10 @@ const Tasks: React.FC  = () => {
                   </td>
                 </tr>
               ))
-            ) : (
-              <NoData/>
-            )}
+            // ) : (
+            //   <NoData/>
```

Comment:
You can render the NoData Component conditionally outside the table as follows:
{tasks?.length > 0 && <NoData}